rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -596,8 +596,14 @@ def __get_dense_features(message: Message) -> Optional[List[Any]]:
 
         tokens = message.get(MESSAGE_TOKENS_NAMES[MESSAGE_TEXT_ATTRIBUTE], [])
         if len(tokens) != len(features):
-            warn_string = f"Number of word embeddings ({len(features)}) does not match number of tokens ({len(tokens)})"
-            raise Exception(warn_string)
+            warnings.warn(
```

Comment:
should warn only if `len(features)>1`? Otherwise it'll be triggered for convert as well

Reply:
Yes, it will be triggered for convert if it is set before the entity extractor. However, I would like to keep it as it is otherwise dense features would just be silently ignored. We should make it clear in the documentation that you cannot use convert here.